### DISCUSSION TITLE
Kulan | Wednesday, 18 December 2024, 8:00 PM

### DISCUSSION THREAD

**Discussion by sharafdin** [dugsiiyeinc/dhaqsoPay#95] [Zoom Links]
Caawa aan soo xaadirno Insha'Allah!

Zoom link:
https://us06web.zoom.us/j/85671421677?pwd=n8xjeceugWPHDHkriPR0fOfnhHIMa6.1

@dugsiiyeinc/dhaqsopay-extension-developers 
@dugsiiyeinc/dhaqsopay-server-developers 
@dugsiiyeinc/dhaqsopay-website-developers

**Comment by AishaKeyf** [comment]
I.a

**Comment by shiinedev** [comment]
Ok bro
